src/TinyCiv.Shared/Dto/PlayerDto.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TinyCiv.Shared.Dto
+{
+    public class PlayerDto
+    {
+        public int Id { get; private set; }
+
+        public PlayerDto(int id)
+        {
+            Id = id;
+        }
+    }
+}
```

Comment:
I think you could have Dtos inside your client project, since server probably (i hope) not use these